src/content/docs/errors-inbox/browser-tab.mdx

```diff
@@ -0,0 +1,147 @@
+To examine JavaScript errors:
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Errors**.
+2. Use the [available options in the UI](#ui-features) to examine what may be causing the errors.
+3. Optional: [Review event trail ](#event-trail) or [stack traces](#stack-trace) in the UI.
+4. Optional: [Query and create dashboards for JS error data](#insights) with the query builder, or [use the API](#api-logs) to monitor or log errors.
+
+## Errors list view  [#list-view]
+
+Start with the page loads errors chart to see at a glance whether there are any unexpected spikes, dips, or patterns with errors in general.
+Correlate any general patterns on the top 5 errors chart to alerts occurring during the same time period.
+* **Dynamic grouping:** The default grouping for error occurrences is based on the error message. You have the option to change the grouping options by any attribute, up to five attributes at a time.
+* **Filtering:** Many New Relic customers instrument [custom attributes](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom). Filtering on a specific custom attribute can be a quick way to cut through the noise of all error occurrences. 
+
+<img
+    title="browser group errors list view"
+    alt="A screenshot depicting the browser group errors list view. "
+    src={errorsinboxBrowserErrorsListView}
+/>
+
+## Error profiles [#JS-profiles] 
+
+Error profiles can help you answer questions such as: Which users are most common across all the error events in the selected time window? or which session is most common across all the error events in the selected time window?
+You can download the attributes as CSV. You can also view the attributes breakdown when you select a row.
+
+<img
+    title="JS error profiles"
+    alt="A screenshot depicting error profiles for JS."
+    src={errorsinboxBrowserErrorProfiles}
+/>
+
+## Errors details view  [#details-view]
+
+On this page, you can dive deep into a specific error group. The detailed view provides in-context details.
+From the detailed view, you can cycle through specific error occurrences using the toggle in the upper right to navigate between the first instance of the error, the last, or any instance between.
+At the top of the details page, you can inspect the **Filtered** and **Grouped By** fields to see how the information was filtered and grouped. You may receive a permalink of a specific occurrence. It's important to know the filters and group-by selections to understand the context of the occurrence.
+
+<img
+    title="browser errors details view"
+    alt="A screenshot depiciting the browser group errors detail view. "
+    src={errorsinboxBrowserErrorsDetailsView}
+/>
+
+## Occurrences tab [#error-group-occurrences]
+
+The **Occurrences** tab includes not only error frequency, occurrence details, and stack traces, it also includes triage information. 
+
+## Triage section [#triage-card]
```

Comment:
I think this should have 3 #: ###Triage section, because it's inside the Occurrences tab